### PR TITLE
feat(aws): add AWS_REGION to aws_prompt_info

### PR DESCRIPTION
**Pull request by sweetpotatoman**
## Standards checklist:

- [x] The PR title is descriptive.
- [x] The PR doesn't replicate another PR which is already open.
- [x] I have read the contribution guide and followed all the instructions.
- [x] The code follows the code style guide detailed in the wiki.
- [x] The code is mine or it's from somewhere with an MIT-compatible license.
- [x] The code is efficient, to the best of my ability, and does not waste computer resources.
- [x] The code is stable and I have tested it myself, to the best of my abilities.

## Changes:
- Only modified the output of `aws_prompt_info`

## Other comments:
Because our company's resource coverage is much more, I hope that the current AWS_REGION can only be displayed, and the display is only changed, and there is no additional `acp()` `asp()`...

### CODE REVIEW

File: plugins/aws/aws.plugin.zsh [hunk 1]
```diff
@@ -152,7 +152,7 @@ compctl -K _aws_profiles asp acp aws_change_access_key
 # AWS prompt
 function aws_prompt_info() {
   [[ -z $AWS_PROFILE ]] && return
-  echo "${ZSH_THEME_AWS_PREFIX:=<aws:}${AWS_PROFILE}${ZSH_THEME_AWS_SUFFIX:=>}"
+  echo "${ZSH_THEME_AWS_PREFIX:=<aws}:${AWS_PROFILE}> <region:${AWS_REGION}${ZSH_THEME_AWS_SUFFIX:=>}"
```
Comment (maksyms) [inline plugins/aws/aws.plugin.zsh:155]: There seems to be a typo here, around `<aws:}$` - please check.

**Review by maksyms — COMMENTED** [review]
Could you also please update `README.md` to reflect that the `aws_prompt_info()` now displays `AWS_REGION` as well as `AWS_PROFILE`?

**Comment by maksyms** [comment]
Thanks for your contribution, @sweetpotatoman! I've started a review - there are very minor points to address, I believe.

File: plugins/aws/aws.plugin.zsh (hunk 1 above)
Comment (sweetpotatoman) [reply plugins/aws/aws.plugin.zsh]: oh, let me change it

**Comment by sweetpotatoman** [comment]
But currently `acp()` and `asp()` do not have `AWS_REGION` set, I set the alias to export the variable by myself. 
Although it does not seem very smart, I think that many people still need to know what area they are working in ...
as shown in the figure:
image

**Comment by sweetpotatoman** [comment]
> Thanks for your contribution, @sweetpotatoman! I've started a review - there are very minor points to address, I believe.

**Comment by maksyms** [comment]
Just a thought: should we update `asp()` and `acp()` to query the profile-configured region and then set `AWS_REGION` accordingly?

**Comment by sweetpotatoman** [comment]
> Just a thought: should we update `asp()` and `acp()` to query the profile-configured region and then set `AWS_REGION` accordingly?
 
Oops...,you are right.
I take a look at how to get more appropriate

**Comment by sweetpotatoman** [comment]
> I have not add region in the original `asp()` and `acp()`. I added a new method.

**Comment by sweetpotatoman** [comment]
@mcornella  Can you add it?

File: plugins/aws/aws.plugin.zsh [hunk 2]
```diff
@@ -143,20 +169,29 @@ function aws_change_access_key() {
   AWS_PAGER="" aws iam list-access-keys
 }
 
+function aws_regions() {
```
Comment (maksyms) [inline plugins/aws/aws.plugin.zsh:172]: Is there a way to query regions from AWS instead of hard-coding them?

**Review by maksyms — COMMENTED** [review]
Tiny small clarifications - but good to go from my perspective after that.

File: plugins/aws/aws.plugin.zsh [hunk 3]
```diff
@@ -21,12 +25,34 @@ function asp() {
   export AWS_DEFAULT_PROFILE=$1
   export AWS_PROFILE=$1
   export AWS_EB_PROFILE=$1
+  # default region set `us-east-1`
```
[the remark this answers is not in the document]
Comment (sweetpotatoman) [reply plugins/aws/aws.plugin.zsh]: > Are these comments necessary?

That's right.

File: plugins/aws/aws.plugin.zsh (hunk 2 above)
Comment (sweetpotatoman) [reply plugins/aws/aws.plugin.zsh:171]: modified!

**Comment by otahontas** [comment]
Hi, are you planning to merge this? Would be useful

**Comment by hans-d** [comment]
what's the status of this PR?

**Comment by maksyms** [comment]
@otahontas @hans-d - I've approved this PR. It is now down to @mcornella to merge.

**Comment by sweetpotatoman** [comment]
Hi, do you still need me to improve? @mcornella

**Comment by sweetpotatoman** [comment]
Hi, do you still need me to improve? again...  @mcornella

**Pull request merged**